docs/workflows/api-token.md

```diff
@@ -112,6 +117,23 @@ curl='curl "https://auth-$(REGION).beyondidentity.com/v1/tenants/$(TENANT_ID)/re
 title="/revoke"
 />
 
+### Revoking token by token_id
+
+In order to revoke a token via management API, you must pass a Bearer
+authentication containing a management API token with the `tokens:delete`
+scope.
+
+You must also know the `TOKEN_ID` of the token you want to revoke. This can be
+returned for example from the token listing.
+
+<MultiLanguageCodeBlock
+curl='curl "https://api-$(REGION).beyondidentity.com/v1/tenants/$(TENANT_ID)/realms/$(REALM_ID)/tokens/$(TOKEN_ID)" \
+-X DELETE \
+-H "Authorization: Bearer $(MANAGEMENT_API_TOKEN)" \
```

Comment:
Should the above curl in `RFC7009 revoke endpoint` also use `MANAGEMENT_API_TOKEN` too?

Reply:
Great catch, this is also a management API token! Plus, we also offer Basic auth version of this endpoint so I'll try adding that too.